cmd/lakefs/cmd/migrate_test.go

```diff
@@ -24,12 +24,25 @@ func TestDoMigrate(t *testing.T) {
 
 	t.Run("not_meets_required_version", func(t *testing.T) {
 		kvStore := kvtest.GetStore(ctx, t)
-		require.NoError(t, kv.SetDBSchemaVersion(ctx, kvStore, kv.InitialMigrateVersion))
+		require.NoError(t, kv.SetDBSchemaVersion(ctx, kvStore, 0))
 		err := cmd.DoMigration(ctx, kvStore, nil, false)
 		require.ErrorIs(t, err, kv.ErrMigrationVersion)
 		version, err := kv.GetDBSchemaVersion(ctx, kvStore)
 		require.NoError(t, err)
-		require.Equal(t, kv.InitialMigrateVersion, version)
+		require.Equal(t, 0, version)
+	})
+
+	t.Run("initial_kv_version", func(t *testing.T) {
+		cfg := config.Config{}
+		cfg.Auth.UIConfig.RBAC = config.AuthRBACSimplified
+		cfg.Auth.Encrypt.SecretKey = "test"
```

Comment:
not sure this is required

Reply:
It is required because the ACL migration initializes an auth service that requires the SecretKey